src/latticpp/ckks/keygen.h

```diff
@@ -63,6 +68,8 @@ namespace latticpp {
 
     RotationKeys genRotationKeysForRotations(const KeyGenerator &keygen, const SecretKey &sk, std::vector<int> shifts);
 
+    RotationKeys genRotationKeysForRotations(const KeyGenerator &keygen, const SecretKey &sk, std::vector<int> shifts, int includeConjugate);
```

Comment:
Nit: We could just keep this API and set the default argument for `includeConjugate` to `false`.